tuning/sft_trainer.py

```diff
@@ -92,7 +91,11 @@ def train(
         tokenizer.add_special_tokens({
             "pad_token": "<pad>",
         })
+    else:
+        logger.error("Unsupported model")
+        exit(-1)
```

Comment:
do we want to exit here?

I think the logics were:
for some models, we want to specially taken care; for the rest, don't do it.

so we might want to consider remove this else block.

Reply:
i wanted to ask @raghukiran1224 about this since I took this from a previous commit in this branch. Bloom Causal LMs have their own tokenizer and if we wnat to continue supporting all causal LMs , I will remove it.

If we only want to officially support granite and llama I will keep it . I think we can keep it open to all Causal LMs Raghu ? (I can remove the else)

Reply:
I removed the else